From 81dc605c1eccddf4cf1c88c42b609da1ca999ba2 Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Fri, 2 Sep 2016 13:02:01 +0800
Subject: [PATCH 1142/5242] MLK-13178-1 arm: imx6qp: turn on vgen3 power up
 pcie ext osc

commit  0b04ce3627e8d2ad4c145290853e07604c37bbbf from
https://source.codeaurora.org/external/imx/linux-imx.git

On imx6qp sabresd rev b board, there is a standalone
external oscilator, used to provided the clks for
imx6qp pcie.
Add one regulator into pcie node, let the ext osc work.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qp-sabresd.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6qp-sabresd.dts b/arch/arm/boot/dts/imx6qp-sabresd.dts
index aa8bc7e..ab27f02 100644
--- a/arch/arm/boot/dts/imx6qp-sabresd.dts
+++ b/arch/arm/boot/dts/imx6qp-sabresd.dts
@@ -119,6 +119,7 @@
 };
 
 &pcie {
+	pcie-bus-supply = <&vgen3_reg>; /* 1.8v pwr up pcie ext osc on revb */
 	reset-gpio = <&gpio7 12 0>;
 	status = "okay";
 };
-- 
1.7.9.5

